modules/platforms/dotnet/Apache.Ignite.Core/Client/Transactions/IClientTransactions.cs

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Core.Client.Transactions
+{
+    using System;
+    using Apache.Ignite.Core.Transactions;
+
+    /// <summary>
+    /// Ignite Thin Client transactions facade.
+    /// </summary>
+    public interface IClientTransactions
```

Comment:
Let's rename to `ITransactionsClient` to be consistent with `ICacheClient`, `IComputeClient`.

Reply:
I think it can be better compared to `IClientCluster` , `IClientClusterNode` and so on. To me `ITransactionsClient` sounds a little bit misleading and clumsy.

Reply:
Actually, `IClientCluster` was missed during the review, so we have an inconsistency now, but I'd like to avoid further inconsistencies.